test/test_tensorboard.py

```diff
@@ -0,0 +1,454 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from common_utils import run_tests
+
+from torch.utils.tensorboard import x2num, SummaryWriter
```

Comment:
If I understand correctly, some of these tests cannot be run if certain optional dependencies are not installed. The test suite should detect if this is the case and skip tests if needed dependencies are not present. You can see examples of this with numpy in existing tests.

Reply:
something like `@unittest.skipIf(...)`?

Reply:
Yep!  There are plenty of examples if you grep around.

Reply:
I am curious that since numpy is an install dependency while installing pytorch, will TEST_NUMPY flag ever been False?